### ISSUE TITLE
Remove Java from the example.krun config and benchmarks

### ISSUE THREAD

**Issue by snim2** [softdevteam/krun#191]
Having Java as a dependency on the examples is problematic for users who just want to quickly see if they can get Krun working. To compile Krun with Java enabled users need to do this:

```bash
 make JAVA_CPPFLAGS='"-I${JAVA_HOME}/include -I${JAVA_HOME}/include/linux"' \
    JAVA_LDFLAGS=-L${JAVA_HOME}/lib ENABLE_JAVA=1
```

rather than just running `make`.

It would be nice to have a variety of language-variants and benchmarks in the examples, to demonstrate how to use the configuration file. It would be preferable to avoid C. It would be preferable if users on most mainstream UNIX platforms did not have to install anything new just to run the examples.

**Comment by vext01** [comment]
Sarah originally chose Java and CPython so we have one interpreted and one compiled language in the example. I like this thinking, but the only other compiled language we have an iteration runner for is C.

**Comment by vext01** [comment]
Can we push forward with this. I'm constantly having to comment out Java in the example, and sometimes I manage to accidentally commit the commenting.

I'm happy to use a C example instead, or to skip Java if the class files are not present.

**Comment by snim2** [comment]
Yes, I'll make a start later today and push something this week...

**Comment by vext01** [comment]
Thanks :)

**Comment by snim2** [comment]
Currently waiting on #200

**Comment by vext01** [comment]
Can I recommend we have a separate example config file for the Java example. Then we get the best of both worlds.

**Comment by snim2** [comment]
OK
